### PR TITLE
mc UAT batch 4

### PR DESCRIPTION
**Pull request by mcolella14**
Jira tickets:

- [DEVHUB-1242](https://jira.mongodb.org/browse/DEVHUB-1242)
- [DEVHUB-1232](https://jira.mongodb.org/browse/DEVHUB-1232)
- [DEVHUB-1110](https://jira.mongodb.org/browse/DEVHUB-1110)
- [DEVHUB-1251](https://jira.mongodb.org/browse/DEVHUB-1251)
- [DEVHUB-1252](https://jira.mongodb.org/browse/DEVHUB-1252)

### CODE REVIEW

**Review by ajhenri — APPROVED** [review]

File: src/pages/author/[...slug].tsx [hunk 1]
```diff
@@ -3,21 +3,18 @@ import { NextSeo } from 'next-seo';
...
 import Card, { getCardProps } from '../../components/card';
 import { getAllContentItems } from '../../service/get-all-content';
 import { ContentItem } from '../../interfaces/content-item';
 import { Grid } from 'theme-ui';
 import { getPlaceHolderImage } from '../../utils/get-place-holder-thumbnail';
 import AuthorSocialButtons from '../../components/author-social-buttons';
+import Breadcrumbs from '../../components/breadcrumbs';
+import { Crumb } from '../../components/breadcrumbs/types';
 
+const crumbs: Crumb[] = [{ text: 'MongoDB Developer Center', url: '/' }];
```
Comment (ajhenri) [inline src/pages/author/[...slug].tsx:17]: So the authors page only has one breadcrumb?

Comment (mcolella14) [reply src/pages/author/[...slug].tsx:17]: Yeah so this is what it is in the design, but it doesn't really make sense to have the Article Name part in there, so I just did the first one. https://devcenter-frontend.devrel.prod.corp.mongodb.com/developer/topics/ also has 1 breadcrumb so I think it should be ok.

**Review by gfunk1230 — APPROVED** [review]

**Comment by gfunk1230** [comment]
@mcolella14 LGTM! and thanks for helping me with this one.

**Review by ajhenri — APPROVED** [review]
Thanks for working on these.

**Pull request merged**